scylla/src/execution/retries/mod.rs

```diff
@@ -0,0 +1,11 @@
+mod default_rp;
+mod downgrading_consistency_rp;
+mod fallthrough_rp;
+mod retry_policy;
+
+pub use default_rp::{DefaultRetryPolicy, DefaultRetrySession};
+pub use downgrading_consistency_rp::{
+    DowngradingConsistencyRetryPolicy, DowngradingConsistencyRetrySession,
+};
+pub use fallthrough_rp::{FallthroughRetryPolicy, FallthroughRetrySession};
+pub use retry_policy::{QueryInfo, RetryDecision, RetryPolicy, RetrySession};
```

Comment:
Commit: "extract and restructure retries module"

🔧 I really don't like this "_rp" in module names.
Wdyt about the following structure:
```
- execution
    - retry_policy
        - mod.rs
        - retry_policy.rs
        - default.rs
        - fallthrough.rs
        - etc...
```

?

Reply:
It's OK to me. Do you agree with the name of module to be `retries` instead of `retry_policy`, as I did in the following commit?

Reply:
Assuming that the parent module is `policies`, and contain `load_balancing` and `speculative_execution` submodules, I think the name `retry` would fit better.

Reply:
Marking as resolved .